common/autoware_ad_api_msgs/routing/msg/Route.msg

```diff
@@ -0,0 +1,2 @@
+std_msgs/Header header
+autoware_ad_api_msgs/RouteBody[<=1] body
```

Comment:
I fixed it because this format cannot contain timestamp when the route is cleared.

Reply:
Are there any problems if we clear the route when empty `segments` are given? Then, I think we can know the stamp.
![image](https://user-images.githubusercontent.com/31987104/183786264-84070a50-68f6-498d-9636-8e218d0b6cca.png)

Reply:
There is no start and goal pose when the route is cleared. So I think this message type is unnatural.

Reply:
I know that, but it's just a trade-off: adding a special value or adding another type for rigor.

Reply:
If @mitsudome-r and @yukkysaito are okay with this type and naming, it's also okay for me.

Reply:
Hmm. In other words, treat an empty section as clearing the route, or add the field like `bool is_cleared`. I think the API needs a message definition that doesn't use incorrectly such as accessing to start and goal when the route is cleared. What do you think? @mitsudome-r @yukkysaito

Reply:
I'll consider the implementation aspects as well.

Reply:
I think there is no need to express emptiness with this message because the route-subscriber nodes can set the data member such as `Route::SharedPtr route_` to be `nullptr` when the RouteState is cleared.
What do you think about it?

Reply:
I understand that this is enough for this message. But that's because this message has a section field and it can have a special meaning for empty value. Other APIs may not be able to do the same. In that case, it will be used an optional field. I would like to avoid this as it breaks design consistency.

Reply:
Let's wait for comments from Mitsudome-san and Saito-san.

Reply:
I prefer @isamu-takagi 's approach. I don't think there is too much cons for adding a new type.

* Route.msg
```
std_msgs/Header header
autoware_ad_api_msgs/RouteBody[<=1] body
```
For the field name, I'd prefer `data`, `route` or `route_body` rather than `body`, but I will leave it up to you whether to change it or not.

* RouteBody.msg
```
geometry_msgs/Pose start
geometry_msgs/Pose goal
autoware_ad_api_msgs/RouteSegment[] segments
```

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1559/commits/a5e08d83dc5ac8935d49feb961365234860ebd37.

- Route.msg
    ```
    std_msgs/Header header
    autoware_ad_api_msgs/RouteData[<=1] data
    ```